src/lib/versions.ts

```diff
@@ -178,6 +178,7 @@ export interface ProcessedNpmInfo {
 	readonly version: Semver;
 	readonly highestSemverVersion: Semver;
 	readonly contentHash: string;
+	readonly distTags: { [tag: string]: string };

```

Comment:
Doesn't appear to be used

Reply:
Which part is not used? `distTags` is needed for line 191.

EDIT: Never mind I see what you mean.